src/sentry/utils/raven.py

```diff
@@ -60,65 +58,21 @@ def send(self, **kwargs):
         if not is_current_event_safe():
             return
 
-        from sentry import tsdb
-        from sentry.coreapi import ClientApiHelper
-        from sentry.event_manager import EventManager
-        from sentry.models import Project
-
-        helper = ClientApiHelper(
-            agent='raven-python/%s (sentry %s)' % (raven.VERSION, sentry.VERSION),
-            project_id=settings.SENTRY_PROJECT,
-            version=self.protocol_version,
+        from sentry.web.api import StoreView
+        headers = {
+            'HTTP_X_SENTRY_AUTH': 'Sentry sentry_key=%s,sentry_version=%s,sentry_client=internal' % (settings.SECRET_KEY, self.protocol_version),
```

Comment:
The only thing I'm unsure about is where to get this auth key from.

Reply:
You can just parse the dsn. There is a function in raven python that does that for you.

Reply:
Hmm, perhaps a dumb question, but which DSN? If I understand correctly, this SentryInternalClient is initialized without a DSN because previously it completely bypassed all the authentication and project selection by calling methods on EventManager directly

Reply:
I'd say we make it configurable just like `SENTRY_PROJECT` which is meant to point to the internal project id. Not sure if we can reliably assume an id of 1 for default out of the box? If not, we can just pick one of the DSNs for this project, which is already the project things are being logged to.